example/android/app/build.gradle

```diff
@@ -88,6 +88,8 @@ android {
     targetSdkVersion rootProject.ext.targetSdkVersion
     versionCode 4138
     versionName "1.0"
+    testBuildType System.getProperty('testBuildType', 'debug')
```

Comment:
Can we please add this:
`// This is for Detox test framework`

So we don't forget why this is needed

Reply:
Done